pom.xml

```diff
@@ -6,11 +6,12 @@
   <parent>
     <groupId>org.kie</groupId>
     <artifactId>kie-parent</artifactId>
-    <version>7.44.0-SNAPSHOT</version>
+    <version>7.45.0-SNAPSHOT</version>
   </parent>
```

Comment:
You shouldn't be using KIE parent anymore. 8.0.0 does not need KIE parent.

Reply:
What should my parent pom be then (if any?)

Reply:
OptaPlanner uses

    <parent>
      <groupId>org.jboss</groupId>
      <artifactId>jboss-parent</artifactId>
      <version>36</version>
      <relativePath/>
    </parent>

To get some basic commonality with middleware. Besides that, just import OptaPlanner BOM and you should be golden.